functional/durable-attestion-sanity-on-localhost/test.sh

```diff
@@ -60,7 +60,11 @@ rlJournalStart
         rlRun "limeWaitForAgentRegistration ${AGENT_ID}"
 
         # create refstat from fake binary_bios_measurements
-        rlRun "python3 /usr/share/keylime/scripts/create_mb_refstate /var/tmp/binary_bios_measurements mb_refstate.txt"
+        if which keylime-policy; then
```

Comment:
It's probably better to use `command -v` instead of `which` (also in the other places it is using it). Other than that, it looks great, thaniks.

Reply:
I'll make the changes and push again